module-2/backend/week_one.md

```diff
@@ -5,30 +5,112 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+
+  post, get, put, delete, options
+
 2. What is Sinatra?
+
+  Sinatra is a MVC based web framework
+
 4. What is MVC?
-5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+
+  Models, Views, Controllers
+
+5. Why do we follow conventions when creating our actions/path names in our Sinatra routes? 
+
+  they accomplish a simplified way to read the path names and make connections between desired actions
+
 6. What types of variables are accessible in our view templates without explicitly passing them?
+
+  the variables can represent the methods in the models, routes 
+
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+  @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+
+  ```ruby
+  get '/horses' do
+  @count = 1
+  @name = Mr. Ed

```

Comment:
this would get the information to the view, but the question is asking how we would pass a **local variable**. When you put the `@` in front of `name`, you're declaring an **instance** variable. To pass a local variable, we would use the following syntax:

```ruby
get '/horses' do
  name = "Mr. Ed"
  erb :index, locals: {name: name}
end
```